src/Auth0Client.ts

```diff
@@ -1257,9 +1212,7 @@ export class Auth0Client {
       tokenResult = await oauthToken(
         {
           ...this.customOptions,
-          ...customOptions,
-          audience,
-          scope,
+          ...options.authorizationParams,
```

Comment:
Same comment here about `this.options.authorizationParams`

Reply:
customOptions is now removed, had to add a `...this.options.authorizationParams` to this call to pull in default custom parameters set on initialization